pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -196,6 +196,7 @@ func (l *LibvirtDomainManager) initializeMigrationMetadata(vmi *v1.VirtualMachin
 	domainSpec.Metadata.KubeVirt.Migration = &api.MigrationMetadata{
 		UID:            vmi.Status.MigrationState.MigrationUID,
 		StartTimestamp: &now,
+		Mode:           v1.MigrationPreCopy,
```

Comment:
We can set this from `MigrationState`. (Useful if we ever don't start with pre-copy)

Reply:
We could, unless we decide it should be set only once the migration is started. Depends how we resolve https://github.com/kubevirt/kubevirt/pull/4016/files#r473326575 

If we decide to not set it there then this value would need to be explicitly set to pre/post/etc here